Source/WebCore/dom/PointerEvent.cpp

```diff
@@ -148,11 +148,25 @@ PointerEvent::~PointerEvent() = default;
 Vector<Ref<PointerEvent>> PointerEvent::getCoalescedEvents() const
 {
 #if ASSERT_ENABLED
+    if (isTrusted()) {
+        auto& names = eventNames();
+        if (type() == names.pointermoveEvent)
+            ASSERT(m_coalescedEvents.size() >= 1, "Trusted pointermove events must have more than zero coalesced events.");
+        else
+            ASSERT(m_coalescedEvents.size() == 0, "Trusted non-pointermove events must have zero coalesced events.");
```

Comment:
I don't have a strong preference, but to balance readability and compliance with https://webkit.org/code-style-guidelines/#zero-comparison, maybe this?

```suggestion
        if (type() == names.pointermoveEvent)
            ASSERT(!m_coalescedEvents.isEmpty(), "Trusted pointermove events must have more than zero coalesced events.");
        else
            ASSERT(m_coalescedEvents.isEmpty(), "Trusted non-pointermove events must have zero coalesced events.");
```

Reply:
I will and already have fixed this in a follow up PR